### PR TITLE
#159 :bug: improve build cache performance on SNAPSHOT plugin updates

### PR DESCRIPTION
**Pull request by d-ryan-ashcraft**

### CODE REVIEW

File: build-parent/pom.xml
Comment (d-ryan-ashcraft) [inline build-parent/pom.xml:168]: I (Informational): context on why this works can be found here: https://codifyiq.substack.com/publish/posts/detail/145767837?referrer=%2Fpublish%2Fposts

We learned this fixing a similar issue on Habushu, so propagating the change here

File: build-parent/pom.xml
Comment (jacksondelametter) [inline build-parent/pom.xml:191]: How does this workaround fix the issue where new versions of SNAPSHOTS are not being invalidated? It looks like the **version.fermenter** is a SNAPSHOT version.

Comment (d-ryan-ashcraft) [reply build-parent/pom.xml:191]: Please see the comment above: https://github.com/boozallen/aissemble/pull/160/files#r1649063234

**Comment by jacksondelametter** [comment]
Looks good to me! Will approve once all 2x SLA tickets are off the board.

**Comment by d-ryan-ashcraft** [comment]
> Looks good to me! Will approve once all 2x SLA tickets are off the board.

Will be more efficient if you just approve it now and I'll hold off on merging for now.

**Review by jacksondelametter — APPROVED** [review]

File: foundation/foundation-archetype/src/main/resources/archetype-resources/pom.xml
Comment (ewilkins-csi) [inline foundation/foundation-archetype/src/main/resources/archetype-resources/pom.xml:68]: A/Q: Since this is in the build-parent, I'd assume downstream projects would pick it up naturally.  Is this not this case and/or is this here as an instructional breadcrumb?

**Review by ewilkins-csi — APPROVED** [review]

**Pull request merged**
